spec_files/system_config.json

```diff
@@ -1,5 +1,10 @@
 {
   "API_PORT": 5000,
+  "RULES_VERSIONS": {
+    "response": 1.0,
+    "callback": 1.0,
+    "validation": 1.0
+  },
```

Comment:
I think we should change the name "RULES_VERSIONS" in system_config and refer it as "CONFIG_VERSIONS". And also we should maintain version for user settings.